tests/inferers/test_zarr_avg_merger.py

```diff
@@ -11,7 +11,9 @@
 
 from __future__ import annotations
 
+import os
 import unittest
+from tempfile import TemporaryDirectory
```

Comment:
I like the use of the `tempfile` module here. There is at least one test file that relies on creating temp files at `tests/testing_data/temp`, the tests at `tests/apps/pathology/test_lesion_froc.py`.
I understand many of the files in `tests/testing_data` are testing assets that can be reused, but it would be nice to implement this `tempfile` module behaviour to all temporary files.